yt_dlp/extractor/rheinmaintv.py

```diff
@@ -0,0 +1,601 @@
+from ..utils import extract_attributes, float_or_none, int_or_none, parse_iso8601
+from .common import InfoExtractor
+
+
+class RheinMainTVIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:www\.)?rheinmaintv\.de/sendungen/(?:[a-z-]+/)*(?P<display_id>[a-z-]+)/vom-(?P<date>[0-9]{2}\.[0-9]{2}\.[0-9]{4})(?:/(?P<serial_number>[0-9]+))?'
+    _TESTS = [{
+        'url': 'https://www.rheinmaintv.de/sendungen/beitrag-video/formationsgemeinschaft-rhein-main-bei-den-deutschen-meisterschaften/vom-14.11.2022/',
+        # 'md5': ...,  # left out because checksum is changing
+        'info_dict': {
```

Comment:
this is not how you write test. See other extractors. Or, delete all the keys except ext/id and run the test. The error message will tell you most of the fields needed

Reply:
That is how I started. Why would you prefer a dummy test (see md5 sum below) over a real one?

What is the purpose of these test cases anyway? I would like to catch any important change of the website, and this includes a lot more than the page title.

Reply:
The changes are cought if the returned `info_dict` has changed values. The suggestion of removing all properties is to generate the correct test case, not for the final test, which must contain all values. If it returns the same data after a change it still works as expected and should not be of concern.

Reply:
So it seems that the test cases are well suited for the final test?!

Reply:
No, it is not - which is obviously why I suggested how to generate them!